cmake/AddInputFileTests.cmake

```diff
@@ -47,6 +51,15 @@ function(add_single_input_file_test INPUT_FILE EXECUTABLE CHECK_TYPE TIMEOUT)
       # Make sure we run the test in the build directory for cleaning its output
       WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
       )
+  elseif("${CHECK_TYPE}" STREQUAL "execute_check_h5")
```

Comment:
perhaps `execute_check_output[_files]` to get this in line with the other names?